awesomplete.js

```diff
@@ -179,6 +185,14 @@ _.prototype = {
 		$.fire(this.input, "awesomplete-open");
 	},
 
+	setListPosition: function () {
```

Comment:
You seem to only be calling this function from one place, so it seems better to inline it there with a comment about what this code does. If in the future it's needed to reposition this list from multiple places *then* it can be moved to a function.